src/Model/Table/SentencesTable.php

```diff
@@ -665,7 +665,7 @@ public function fields($what = [])
     public function contain($what = [])
     {
         $audioContainment = function (Query $q) use ($what) {
-            $q = $q->select(['id', 'external', 'sentence_id']);
```

Comment:
I think this should be moved to the `if (isset($what['sentenceDetails'])) {` block below in a way that adds `created` and `modified` fields to the `select` operation. This way we only retrieve these extra fields on sentence pages (as opposed to retrieving them everywhere a sentence block is displayed).

Reply:
See commit ba8e93d